olive/cli/quantize.py

```diff
@@ -75,6 +82,15 @@ def register_subcommand(parser: ArgumentParser):
         # dataset options
         add_dataset_options(sub_parser, required=False, include_train=False, include_eval=False)
 
+        quarot_group = sub_parser.add_argument_group(
+            "quarot options, required for quantization using QuaRot algorithm."
+        )
+        quarot_group.add_argument("--rotate", action="store_true", help="Apply QuaRot/Hadamard rotation to the model.")
+
+        using_group = quarot_group.add_mutually_exclusive_group(required=False)
+        using_group.add_argument("--rtn", action="store_true", help="Quantize weights using RTN.")
```

Comment:
Since, one of these are required, I think this could be made into a single option like `--quarot_quant_algo` with a default value if needed (like rtn).